.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Submit a bug request
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+*Please do not include your AWS account or credentials as part of the bug request.*
+
+**Describe the bug**
+ Please describe the behavior that you are seeing with as many details as possible including logs from AGC, Cloudwatch snippets and Cloudformation errors if relevant. 
+**To Reproduce**
+ Steps to reproduce the behavior:
+
+1. “agc account activate”
+2. “agc context deploy —all -v”
+3. See error
+
+**Logs**
+The logs that where obtained by running the commands here. Please run the command with "-v" so that we can see the verbose logs.
+**Expected behavior**
+ A clear and concise description of what you expected to happen.
+**Screenshots**
+ If applicable, add screenshots to help explain your problem.
+**Additional context**
+ Add any other context about the problem here.
```

Comment:
You're swapping between an actual example involving `context deploy` and  descriptions of what should go where. I'd stick with just placeholders.